### DISCUSSION TITLE
Floret vectors on spacy 3.5

### DISCUSSION THREAD

**Discussion by jkgenser** [explosion/spaCy#12338] [Help: Best practices]
Hello - 

I am looking at incorporating floret vectors into my pipeline in order to be robust to miss-spellings on an NER task. I am experimenting with using these vectors below although it seems they are set to work with spacy 3.4 and I'm using 3.5.

I receive a warning when loading them but despite this my model seems to be training reasonably well. Are there any weird gotchas I should look out for?

https://github.com/explosion/spacy-vectors-builder/releases/download/en-3.4.0/en_vectors_floret_lg-0.0.1-py3-none-any.whl

**Comment by adrianeboyd** [comment]
There shouldn't be any compatibility problems with floret vectors across v3.x models/versions.
